DESIGN.md

```diff
@@ -138,6 +145,14 @@ Installing a specific snapshot from a swift version development branch
 
 `swiftly install 5.5-snapshot-2022-1-28`
 
+##### Installing the version from the `.swift-version` file
+
+A package could have a swift version file that specifies the recommended toolchain version. A swiftly install with no version will search for a version file and install that version.
+
+`swiftly install`
+
+If no swift version file can be found then the installation fails indicating that it couldn't fine the file.
```

Comment:
Very small nit, but could we use
> If no .swift-version file ...

Rather than 
> swift version file

Just to be consistent, so users don't conflate the "swift version file" with something else in the Swift ecosystem. This way we explicitly mention the file in question.  This is how it's written in the _"Installing the version from the `.swift-version` files"_ section